CHANGELOG.md

```diff
@@ -6,7 +6,7 @@ v0.8.6 - 2019-09-
 
 ### Features and improvements ###
 
-- Updated compiler support for DMD 2.087.1 and LDC 1.6.0 - [pull #2312][issue2312], [pull #2330][issue2330]
+- Updated compiler support for DMD 2.087.1 and LDC 1.7.0 - [pull #2312][issue2312], [pull #2330][issue2330]
```

Comment:
Typo? (LDC 1.**1**7.0)

Reply:
Saw this too late, after the auto-merge went through, but fixed now in 258fbfe17ad015f42c216057f1f4ba4536c85162